upgrade-instructions.md

```diff
@@ -14,4 +14,70 @@ Note: this section is in progress. It will be updated during all the development
 
 Change list:
 
-(No changes yet.)
\ No newline at end of file
+(No changes yet.)
+
+## 0.6.3 version
+
+Change list:
+
+- [Improve indexing of email address domains](#improve-indexing-of-email-address-domains)
+
+### Improve indexing of email address domains
```

Comment:
I think that upgrade instruction should be part of apache james and not necessary here..?

Reply:
We will need documentation upgrade instructions for `JAMES-3827 Backport to ElasticSearch 6.x` anyway.

Reply:
Usually putting them on the changelog but... it's true that we don't use this upgrade-instructions in this project until now. Deserves a discussion :)

Reply:
No not in james as we migrated from OpenSearch.

In TMail it is different as we have an intermediate OpenSearch release, which thus need an upgrade instruction.